pants.ci.toml

```diff
@@ -6,7 +9,7 @@ pantsd = false
 # Limit the maximum number of concurrent processes. Change this
 # to a number that makes sense for your CI setup, based on
 # the number of cores/threads.
-process_execution_local_parallelism = 4
+process_execution_local_parallelism = 2
```

Comment:
Why not just omit this entirely? The default is probably sound. Or at least comment that the default is equal to the number of cores reported by the system.

Reply:
Because this is likely something people will want to tune, and we want to highlight that it's an option.